### PR TITLE
update to lates exchangelib version in ewsv2, remove pwsh

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9764) 
[Bump exchangelib in EWSv2 docker](https://jira-dc.paloaltonetworks.com/browse/CIAC-10403)

## Docker image link
https://github.com/demisto/dockerfiles/pull/27234

## Description
- updating docker to new based python debian version
- removing temporary changes that were relevant to previous exchangelib version since it was bumped.
- removing compliance commands from yml and python

### CODE REVIEW

File: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_7.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Updated the Docker image to: *demisto/py-ews:5.3.0.93845*.
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_7.md:6]:
```suggestion
- Updated the Docker image to: *demisto/py-ews:1.0.0.97376*.
```

File: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_7.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Updated the Docker image to: *demisto/py-ews:1.0.0.97376*.
+- Adding support in version ***Exchange 2013 SP1***.
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_7.md:7]:
```suggestion
- Adding support for version ***Exchange 2013 SP1***.
```

File: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_7.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Updated the Docker image to: *demisto/py-ews:1.0.0.97376*.
+- Adding support for version ***Exchange 2013 SP1***.
+- Deprecated and deleted the commands that can be used from Security And Compliance V2 instead:
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_7.md:8]:
```suggestion
- Deleted the following commands . Use **Security And Compliance V2** instead:
```

**Review by JudahSchwartz — APPROVED** [review]

File: Tests/conf.json [hunk 4]
```diff
@@ -1608,11 +1608,7 @@
             "fromversion": "4.5.0",
             "memory_threshold": 150
         },
-        {
-            "integrations": "EWS v2",
-            "playbookID": "get_original_email_-_ews-_test",
-            "instance_names": "ewv2_regular"
-        },
```
Comment (sapirshuker) [inline Tests/conf.json:1615]: Why?

**Comment by JudahSchwartz** [comment]
needs force like https://github.com/demisto/content/pull/32613 for https://jira-dc.paloaltonetworks.com/browse/CIAC-9721
same problem

**Pull request merged**